app/views/planning_applications/review_form.html.erb

```diff
@@ -14,9 +14,7 @@
     </strong>
   </p>
   <p class="govuk-body">
-    It adheres to the limitations and conditions of Part 1 Class A and F,
-    Schedule 2 of The Town and Country Planning (General Permitted Development)
-    Order 2015 (as amended).
+    It adheres to the limitations and conditions of <%= @planning_application.public_comment %>
```

Comment:
Reading what Martyn has said on the ticket, I think you want to remove "It adheres to the limitations and conditions of" too.

Reply:
I sent him a screenshot before pushing the PR up and he said it looked good

Reply:
I've just checked with Martyn: "the whole paragraph should be the planning officers comment"